### ISSUE TITLE
a11y: Color Contrast - Create User

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13736]
Insufficient color contrast for informative image

## Actual result

"The informative image present on the page has an insufficient color contrast ratio with their background
making it difficult for low vision users to discern meaningful graphics that have insufficient contrast."

## Instances

"Foreground Text Color:  #F87676
Background Text Color: #FEE4E4
Ratio: 2.22:1 
Main Content
- Close Button (Available after adding invalid values in the input fields) 

Foreground Text Color:  #3D98D3
Background Text Color: #FEE4E4
Ratio: 2.62:1 
Main Content
- Close Button (on hover ) (Available after adding invalid values in the input fields) "

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
- Cancel and Show Button (on default state)
- Release Notes
- Username, Display Name, Description, New Password  and Confirm Password input fields ( on hover and focus) 
- Show and Hide link text. 
- Suggest a password (available after activating Generate a random password checkbox) 

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1 
Main Content
- e.g. jsmith, e.g. John Smith, and  e.g. This account is for John Smith

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Create Button (available after adding values in the input fields ) 

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1 
Main Content
- e.g. jsmith, e.g. John Smith, and  e.g. This account is for John Smith

Foreground Text Color: #898991
Background Text Color: #FFFFFF 
Ratio: 3.47:1 
Main Content
-Administrators have full control over the entire installation and all resources in all cluster
- Restricted Admins have full control over all resources in all downstream clusters but no access to the local cluster
- Standard Users can create new clusters and manage clusters and projects they have been granted access to
- User-Base users have login-access only
- Allows the user to enable, configure, and disable all Authentication provider settings. 
and so on. 

Foreground Text Color:  #FFFFFF 
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel and Create Button (on focus) 

Foreground Text Color: #F64747
Background Text Color: #FEE4E4
Ratio: 3.16:1 
Main Content
- Password must be at least 12 characters,Passwords do not match (available after adding invalid values in the password input field ) 

Foreground Text Color:  #FFFFFF 
Background Text Color: #F64747
Ratio: 3.55:1 
Main Content
- Error Button (available after adding invalid values in the  input fields) 
    "

## Expected results

The contrast ratio between the key part of the image and its adjacent colors should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF 
> Ratio: 1.78:1 
> Main Content
> - e.g. jsmith, e.g. John Smith, and  e.g. This account is for John Smith
> 
> Foreground color: #2F68DF
> Background color: #EDF2FC
> Contrast ratio: 4.4:1
> - Cancel (On hover on focus )
> 
> Foreground Text Color: #6C6F76
> Background Text Color: #FFFFFF 
> Ratio: 3.3:1 
> Main Content
> -Administrators have full control over the entire installation and all resources in all cluster
> - Restricted Admins have full control over all resources in all downstream clusters but no access to the local cluster
>- Standard Users can create new clusters and manage clusters and projects they have been granted access to
> - User-Base users have login-access only
> - Allows the user to enable, configure, and disable all Authentication provider settings. 
and so on. 
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 211
Issue 212 - Fixed
